backend/src/test/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtilsTest.java

```diff
@@ -135,4 +138,66 @@ void invalidSpecimenSNOMED() {
     assertThat(validateSpecimenType(specimenType, Map.of("oral saliva sample", "000111222")))
         .hasSize(1);
   }
+
+  @Test
+  void validDateFormat() {
+    var validDates = new ArrayList<ValueOrError>();
+    validDates.add(new ValueOrError("01/01/2023", "date"));
+    validDates.add(new ValueOrError("1/1/2023", "date"));
+    validDates.add(new ValueOrError("1/01/2023", "date"));
+    validDates.add(new ValueOrError("01/1/2023", "date"));
+    validDates.add(new ValueOrError("1/31/2023", "date"));
+    validDates.add(new ValueOrError("12/01/2023", "date"));
+    for (var date : validDates) {
+      assertThat(validateDateFormat(date)).isEmpty();
+    }
+  }
+
+  @Test
+  void invalidDateFormat() {
+    var validDates = new ArrayList<ValueOrError>();
```

Comment:
I would rename the variable to `invalidDates`

Reply:
Good catch on the copy and paste!